### DISCUSSION TITLE
[Feature Request] Google Cloud Monitoring MQL Query Type Variable

### DISCUSSION THREAD

**Discussion by ChristopherWriight** [grafana/grafana#64350] [Data Sources]
The GCM plugin doesn't allow fully custom query variables for chained variables. The current setup only allows for dropdown selections.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82799. Feel free to continue the discussion around this there. Thank you!
